### PR TITLE
[16.0][MIG] helpdesk_mgmt migration to 16.0

### PR DESCRIPTION
**Pull request by zuher83**
Hi
Migration of **helpdesk_mgmt** from 15.0 to 16.0
I have been testing for a few days and everything seems to work fine in a new Odoo 16.0 installation

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks for the contribution.

Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0.

If the jump is between several versions, you have to modify the source branch in the main command to accommodate it to this circumstance.

**Comment by zuher83** [comment]
Sorry, I'm going to start the job again using the OCA method.
Do I close this PR and #441 to propose a new PR ?

**Comment by pedrobaeza** [comment]
You can restart your work locally with the same branch name, and do git push -f

**Comment by zuher83** [comment]
I need to fix test ...

**Comment by zuher83** [comment]
@pedrobaeza I think Helpdesk Management is ready to merge 🙂 if you have any suggestions ...

**Comment by pedrobaeza** [comment]
/ocabot migration helpdesk_mgmt

Now you need to enroll at least 2 people to review your work:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#review

You can review other PRs, and ask in exchange that they review yours.

**Comment by davide445** [comment]
Hi we are interested in using this on our corporate 16.0 community installation, will be safe to use it now?
There is any ETA for the official merge?

**Comment by pedrobaeza** [comment]
Try yourself here in runboat and make a functional review:

https://odoo-community.org/resources/review

**Comment by davide445** [comment]
> Try yourself here in runboat and make a functional review:
> 
> https://odoo-community.org/resources/review

Apologize for asking trivial questions, you mean testing [this](https://github.com/zuher83/helpdesk/tree/16.0-mig-helpdesk_mgmt) using this [tool](https://github.com/sbidoul/runboat)?

Our infra didn't use Kubernets so will be longer than needed to setup I think, if you can point me what release we can install we can just create a copy of our current production env.

**Comment by pedrobaeza** [comment]
You shouldn't do anything on your side, except trying functionally following the instructions. Please see the link I put.

**Comment by davide445** [comment]
Sorry seems I'm too dumb to contribute, due I have read the guide and I didn't find any "contribution" item, only Issues or Pull requests, neither of has checks or links to runboat.
Will wait for the official merge.

**Comment by pedrobaeza** [comment]
You have to do it in this pull request. If you are writing here, you are accessing to "the contribution". Let me put a mini-video about how to test it:

Peek 27-03-2023 16-20

**Review by davide445 — APPROVED** [review]
Seems clean and working. The reporting pivot is nice. I will prefer to have directly the integration with the Projects due this will be needed for tracking issues related to specific customers, but as far I know there is another app doing this.

File: helpdesk_mgmt/static/description/icon.png
Comment (pedrobaeza) [inline helpdesk_mgmt/static/description/icon.png:1]: Why touching the icon?

**Comment by davide445** [comment]
As written to me seems fine in testing environment, but my goal is testing it in our Odoo 16 CE installation. There is any ETA for the PR approval.

File: helpdesk_mgmt/static/description/icon.png
Comment (pedrobaeza) [reply helpdesk_mgmt/static/description/icon.png:1]: cc @zuher83

**Comment by zuher83** [comment]
@davide445 It depends on reviewers of the pull request... I use it in production, and it works very well, the changes between 15.0 and 16.0 are minor

**Review by destroT — APPROVED** [review]
LGTM

File: helpdesk_mgmt/static/description/icon.png
Comment (pedrobaeza) [reply helpdesk_mgmt/static/description/icon.png:1]: Is this the icon format for v16? Please check it in https://ivantodorovich.github.io/odoo-icon/

**Comment by AntoniRomera** [comment]
@OCA/helpdesk-maintainers any one can merge?

**Comment by pedrobaeza** [comment]
My question about the icon is still opened

**Comment by AntoniRomera** [comment]
My bad, I think is corrected

**Comment by pedrobaeza** [comment]
Thanks for the change in the icon. With the icon builder, it's possible to put the same color if you want. Anyway, please squash the 3 migration commits into one and I'll finally merge this.

**Comment by zuher83** [comment]
@pedrobaeza changed icon to same color :) and squashed for finally merge

**Comment by mgielissen** [comment]
What about /migration for OpenUpgrade?

**Pull request merged**
